public/index.html

```diff
@@ -73,6 +73,6 @@
 
     <script src="https://www.google.com/recaptcha/api.js?render=explicit" async defer></script>
     <script type="text/javascript" src="https://apis.google.com/js/api.js"></script>
-    <script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs"></script>
+    <script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs" data-app-key="ukbw5572vxp6oib"></script>
```

Comment:
How did we get the `data-app-key`

Reply:
@tnagorra it was defined in a utils file previously